File: DofusSpa/src/app/_models/harness.ts
import { HarnessRecipe } from "./harness-recipe";

export interface Harness{
    id : number,
    ankamaID : number,
    name : string,
    url : string,
    imgUrl : string,
    level : number,
    type: string,
    description : string,
    recipes : HarnessRecipe[],
}

File: DofusApi/Models/Harness.cs
using System.Collections.Generic;

namespace DofusApi.Models
{
    public class Harness
    {
        public int ID { get; set; }
        public int AnkamaID { get; set; }
        public string Name { get; set; }
        public int Level { get; set; }
        public string Type { get; set; }
        public string ImgUrl { get; set; }
        public string Url { get; set; }
        public string Description { get; set; }
        public ICollection<HarnessRecipe> Recipes { get; set; }
        
    }
}